### ISSUE TITLE
Issue with relative path resolution in meta-conduct?

### ISSUE THREAD

**Issue by jkosciessa** [datalad/datalad-metalad#253]
While trying to run through a datacat tutorial with @jsheunis, I stumbled upon a slight inconsistency in how paths appear to be resolved by the `meta-conduct` command. When specifying a relative path (working directory is `/Users/kosciessa/hackathon2022`) to the datalad dataset, the specified relative path appears to be duplicated before being appended to the working directory, resulting in the error shown below.

```
datalad -f json meta-conduct ./fairly-big-catalog-workflow/conduct_pipelines/extract_file_metadata.json \
    traverser.top_level_dir=./eegmp/eegmp_data/ \
    traverser.item_type=file \
    extractor.extractor_type=file \
    extractor.extractor_name=metalad_core \
    | jq -c '.["pipeline_data"]["result"]["metadata"][0]["metadata_record"]' >> ./eegmp_cat/eegmp_data_filemeta.txt
```

```
[ERROR] NoDatasetFound(No installed dataset found at /Users/kosciessa/hackathon2022/eegmp/eegmp_data/eegmp/eegmp_data) (NoDatasetFound) 
```

Everything is running smoothly when I specify the absolute path.

```
datalad -f json meta-conduct /Users/kosciessa/hackathon2022/fairly-big-catalog-workflow/conduct_pipelines/extract_file_metadata.json \
    traverser.top_level_dir=/Users/kosciessa/hackathon2022/eegmp/eegmp_data/ \
    traverser.item_type=file \
    extractor.extractor_type=file \
    extractor.extractor_name=metalad_core \
    | jq -c '.["pipeline_data"]["result"]["metadata"][0]["metadata_record"]' >> /Users/kosciessa/hackathon2022/eegmp_cat/eegmp_data_filemeta.txt
```

Could this be down to an error in the relative path resolution of the code?

**Comment by christian-monch** [comment]
@jkosciessa: Thx for catching this. I just released version 0.3.3 which fixes the bug

**Comment by christian-monch** [comment]
Fixed in release 0.3.3
